### PR TITLE
Re-buil sql scripts:

### PR DESCRIPTION
**Pull request by RomanKlishch**
- changed all scripts
- added new scripts

### CODE REVIEW

File: src/main/resources/db/migration/V11__another_link_schema.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE another_links
+(
+    another_link_id   SERIAL PRIMARY KEY,
+    another_path      VARCHAR(500),
```
Comment (AlexKievDeveloper) [inline src/main/resources/db/migration/V11__another_link_schema.sql:4]: another_path - "другой путь" 
Think better will be use just "link"

File: src/main/resources/db/migration/V1__language_schema.sql [hunk 2]
```diff
@@ -1,5 +1,5 @@
 CREATE TABLE language
```
Comment (AlexKievDeveloper) [inline src/main/resources/db/migration/V1__language_schema.sql:1]: languageS

File: src/main/resources/db/migration/V10__audio_schema.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE audios
+(
+    audio_id          SERIAL PRIMARY KEY,
+    audio_path        VARCHAR(500),
```
Comment (AlexKievDeveloper) [inline src/main/resources/db/migration/V10__audio_schema.sql:4]: Why VARCHAR length is 500? Not 200?

File: src/main/resources/db/migration/V3__statuses_schema.sql [hunk 4]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE statuses
+(
+    status_id SERIAL PRIMARY KEY,
+    status    INTEGER
```
Comment (AlexKievDeveloper) [inline src/main/resources/db/migration/V3__statuses_schema.sql:4]: Why Integer not Varchar?

File: src/main/resources/db/migration/V8__youtube_schema.sql [hunk 5]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE youtube_links
+(
+    youtube_link_id   SERIAL PRIMARY KEY,
+    youtube_path      VARCHAR(500),
```
Comment (AlexKievDeveloper) [inline src/main/resources/db/migration/V8__youtube_schema.sql:4]: I suggest renaming youtube_path to youtube_link

File: src/test/resources/testDB/migration/R__2setUpUsers.sql [hunk 6]
```diff
@@ -1,4 +1,19 @@
...
+--     user_id     SERIAL PRIMARY KEY,
+--     firstName   VARCHAR(40)        NOT NULL,
+--     lastName    VARCHAR(40),
+--     login       VARCHAR(50) UNIQUE NOT NULL,
+--     email       VARCHAR(50) UNIQUE NOT NULL,
+--     password    VARCHAR(200)       NOT NULL,
+--     salt        VARCHAR(200)       NOT NULL,
+--     language_id INTEGER            NOT NULL,
+--     FOREIGN KEY (language_id) REFERENCES language (language_id)
+-- );
```
Comment (AlexKievDeveloper) [inline src/test/resources/testDB/migration/R__2setUpUsers.sql:19]: You left some commented code)

File: src/test/resources/testDB/migration/R__1setUpLanguages.sql [hunk 7]
```diff
@@ -1,2 +1,10 @@
...
+INSERT INTO languages(language_id, name) VALUES (1,1);
+INSERT INTO languages(language_id, name) VALUES (2,2);
+
+
+
+-- CREATE TABLE language
+-- (
+--     language_id SERIAL PRIMARY KEY,
+--     name    INTEGER
+-- );
```
Comment (AlexKievDeveloper) [inline src/test/resources/testDB/migration/R__1setUpLanguages.sql:10]: You left some commented code)

File: src/test/resources/testDB/migration/R__3setUpStatus.sql [hunk 8]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO statuses(status_id, status) VALUES (1,1);
+INSERT INTO statuses(status_id, status) VALUES (2,2);
+
+
+
+-- CREATE TABLE statuses
+-- (
+--     status_id SERIAL PRIMARY KEY,
+--     status    INTEGER
+-- );
```
Comment (AlexKievDeveloper) [inline src/test/resources/testDB/migration/R__3setUpStatus.sql:10]: You left some commented code)

**Review by OleksandrGubariev — APPROVED** [review]

**Review by AlexKievDeveloper — APPROVED** [review]

**Pull request merged**